assets/js/components/adminbar/AdminBarApp.js

```diff
@@ -75,6 +79,30 @@ export default function AdminBarApp() {
 						mdc-layout-grid__cell--align-middle
 					">
 						<div className="mdc-layout-grid__inner">
+							{ featureFlags.widgets.adminBar.enabled && (
+								<Fragment>
+									{ analyticsModuleConnected && (
+										<Fragment>
+											<AdminBarUniqueVisitors />
+											<AdminBarSessions />
+										</Fragment>
+									) }
+
+									{ ! analyticsModuleConnected && (
+										<div className="
+											mdc-layout-grid__cell
+											mdc-layout-grid__cell--span-6-desktop
+											mdc-layout-grid__cell--span-4-tablet
+										">
+											<AnalyticsInactiveCTA />
+										</div>
+									) }
+
+									<AdminBarImpressions />
+									<AdminBarClicks />
```

Comment:
It looks like these need to go before the Analytics components rather than after.

From the current version:
![image](https://user-images.githubusercontent.com/1621608/103541744-87825200-4ea4-11eb-9f2d-d5efcf169c75.png)

Reply:
Updated @aaemnnosttv